usecases/partner_usecase.go

```diff
@@ -63,6 +65,7 @@ func (usecase *PartnerUsecase) CreatePartner(
 		tx repositories.Executor,
 	) (models.Partner, error) {
 		partnerId := uuid.New().String()
+		partnerCreateInput.Bic = strings.TrimSpace(strings.ToUpper(partnerCreateInput.Bic))
```

Comment:
For now and since it's an internal endpoint, I'm doing minimal data cleaning rather than full input validation